tests/test_validation.py

```diff
@@ -0,0 +1,28 @@
+import base64
+
+import h5py
+import pytest
+
+from neurons.old.protocol import TextTo3D
```

Comment:
This process should be protocol agnostic. Just use binary data

Reply:
Removed usage of TextTo3D from code in validation